namespace Common.Exceptions
{
    public class MensajeReferencia
    {
        public const string RegistroNoEncontrado = "El registro no ha sido encontrado";
        public const string CampoRequerido = "El campo '{0}' es requerido para poder continuar";
        public const string ErrorServidor = "Ocurrió un error interno.";
        public const string CreacionExitosa = "Registro creado con exito";
        public const string ActualizacionExitosa = "Registro actualizado con éxito";
        public const string EliminacionExitosa = "Registro Eliminado con éxito";
        public const string Conflicto = "A ocurrido un conflicto";
        public const string NoEncontrado = "Registro no encontrado";
        public const string NoImplementado = "No Implementado.. Estamos trabajando";
        public const string NoAutorizado = "No posee los permisos necesario para realizar esta petición";
    }
}